src/jemalloc.c

```diff
@@ -3040,142 +2987,11 @@ free_default(void *ptr) {
 	}
 }
 
-JEMALLOC_ALWAYS_INLINE bool
```

Comment:
I assume these are all moved around, w/o any real changes made. Is that the case?

Reply:
Yes, no changes made to the codes